nss-day-cohort-47/reactive-nutshell-the-dial-up-dudes
# Message Sort

# Description
Messenger now displays messages directed to user, public messages, and users own messages. 
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Click messages. Try sending message @ another user. Edit that message to @ another user. The receiver ID should change when you edit it. Only the person you sent the message to should be able to see the message. If you send a message without an @ then it will be public. 
# Checklist:
